packages/ocaml-variants/ocaml-variants.5.1.2+trunk/opam

```diff
@@ -7,7 +7,7 @@ homepage: "https://ocaml.org"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
```

Comment:
removing 5.1.1+trunk will cause problem for the people not using gpatch on macOS. Could you instead use `available: false` and create a new package for 5.1.2+trunk?

Reply:
Done, thanks !